docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -20,11 +20,11 @@ See the [Terraform installation guide](/reference/terraform-plugin#install) to i
 
 Using Terraform, you will deploy a simple guestbook application backed by AWS DynamoDB Table. The terraform configuration file will use both the Layer0 and AWS Terraform providers, to deploy the guestbook application and provision a new DynamoDB Table.
 
-## Part 1: Clone the Layer0 examples repository
+## Part 1: Clone the guides repository
 
-Run this command to clone the `quintilesims/layer0-examples` repository. Once you have cloned the repository, navigate to `.terraform-beyond-layer0/example-1` folder for the rest of this example.
+Run this command to clone the `quintilesims/guides` repository. Once you have cloned the repository, navigate to `.terraform-beyond-layer0/example-1` folder for the rest of this example.
```

Comment:
navigating to `.terraform-beyond-layer0/example-1` will fail in this example as we're not in the `/guides` dir after `clone`. I'd suggest using explicit paths for these types of instructions.